### DISCUSSION TITLE
ID3 tag: year

### DISCUSSION THREAD

**Discussion by kp-org** [jcorporation/myMPD#900] [Ideas]
Hi,

it would be nice to have the id3 tag year information available at several places:

- at the playback page, configurable from the drop-down tag list
- at the browse page for "all albums" and "all albums for artist" view, maybe as a third line and configurable (display or not)

Perhaps there are more places.

**Comment by jcorporation** [comment]
https://mpd.readthedocs.io/en/latest/protocol.html#tags

MPD does not support a tag named `year`, do you mean the `date` tag?

> at the playback page, configurable from the drop-down tag list

All supported and enabled tags can be selected in the drop-down tag list in the playback card and for tables.

Configurable tags for the album overview is a good idea.

**Reply by kp-org** [reply]
However the field is called internal, from https://id3.org/: "This data commonly contains the Artist name, Song title, Year and Genre of the current audio file." And yes, in mpd it seems to be the "date" field.

In mpd the date field is available, but there isn't a date tag in mympd's playback page available:
![image](https://user-images.githubusercontent.com/13482381/203251885-1969f9a6-e2ab-4d59-8168-15c0f651293d.png)

From my mpd.conf:
metadata_to_use "artist,album,title,track,name,genre,date,composer,performer,disc"
metadata_to_use "+comment"

Am I missing something?

**Reply by jcorporation** [reply]
You must enable the tag in the myMPD settings.

**Comment by kp-org** [comment]
Got the missing piece, thanks.

**Comment by avalx** [comment]
Hmm, has this changed in latest releases? It used to work fine but with the latest 10.2.0 version the date/year field in the Browse view is empty (-). Also when I browse by Date I can see all the years but when I click any of those the response is 'Empty list'?! On the other hand I can see Date in the Album view as well as in the Queue and Playback views.

**Comment by jcorporation** [comment]
This is indeed a bug in a recent commit. Fix is on the way.

**Comment by avalx** [comment]
OK, thanks

**Reply by jcorporation** [reply]
Is now fixed in the v10.2.0 branch. Note: that is a unstable development branch, but I appreciate testers.
